book/src/types/rust_types.md

```diff
@@ -142,7 +142,7 @@ them in the [aliases chapter](./rust_types/alias.md).
 The `BoundVar` variant represents some variable that is bound in
 an outer term. For example, given a term like `forall<X> {
 Implemented(X: Trait) }`, the `X` is bound. Bound variables in chalk
-(like rustc) use de bruijin indices (See below).
+(like rustc) use de bruijn indices (See below).
```

Comment:
We should change this to "De Bruijn indices"

Reply:
Changed all occurrences to De Bruijn